src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.html

```diff
@@ -49,7 +49,7 @@
                 <i class=" icon">+91-</i>
               </div>
             </div>
-            <label class="ui basic blue error label pt-0" *ngIf="signUpForm.controls.password.touched && signUpForm.controls['password'].errors">{{resourceService.frmelmnts.lbl.validPassword}}</label>
+            <label class="ui basic blue error label pt-0" *ngIf="signUpForm.controls.password.touched && isPasswordError">{{passwordErrorMsg}}</label>
```

Comment:
We dont need "isPasswordError" field. Remove this field & relay of "passwordErrorMsg" filed itself. Based on length show the message.